backend/register_lambda.py

```diff
@@ -4,6 +4,9 @@
 
 def lambda_handler(event, context):
     params = event['queryStringParameters']
+    if 'email' not in params or 'dropbox_oauth_token' not in params:
+        print('Missing info for registration')
+        exit()
```

Comment:
Wouldn't this actually not be a graceful quit? We would just return nothing to the frontend here right? We also aren't catching cases where ddb fails and any other generic errors